scripts/redis/scan_redis.py

```diff
@@ -40,16 +38,30 @@ def get_size(client, key, key_type):
     return size
 
 
-def run():
+@click.command()
+@click.option('--file', 'file_name', default='redis-stats.log')
+@click.option('--match', default=None)
```

Comment:
Add a check that prevents setting ttl if match `is None` or empty string. We probably don't want to accidentally trash an entire redis, and if we did, that redis should have had a global ttl set up instead.